### DISCUSSION TITLE
Word count

### DISCUSSION THREAD

**Discussion by udfed** [toeverything/AFFiNE#6172] [Ideas]
### Description

I would like to have ​word count (selected / total) for each page and each note card in edgeless mode.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

[a comment by an automated account is not shown]

**Reply by ZocDock** [reply]
It's definitely something I am missing right now

**Reply by mawnir** [reply]
yes please

**Reply by wangjc683** [reply]
same here, definitely need the word count as a literature worker myself.

**Reply by Sigabiel** [reply]
same

**Reply by NotTradingAdvice** [reply]
Much needed

**Comment by CX330Blake** [comment]
100%. Any update?
